### ISSUE TITLE
Menu does not focus if first item is disabled and is not focusable for keyboard (a11y).

### ISSUE THREAD

**Issue by rgthree** [angular/material#9972]
**Actual Behavior**:
 -  `What is the issue? *` When opening a menu list whose first item is disabled, the menu does not focus. This results in the menu not being usable with only a keyboard as arrows/tabs move around behind the menu.
 -  `What is the expected behavior?` The menu would focus the first item that can be focused and works as expected otherwise.

**CodePen** (or steps to reproduce the issue): *
 -  `CodePen Demo which shows your issue:` http://codepen.io/rgthree/pen/dOyaVj
 -  `Details:` In the demo

**Angular Versions**: *
 -  `Angular Version:`  1.5.5/angular.js, 1.5.5/angular-aria.js, 1.5.5/angular-animate.js
 -  `Angular Material Version:` 1.1.0/angular-material.min.js

**Additional Information**:
 -  `Browser Type: *` Chrome
 -  `Browser Version: *` 56.0.2902.0 (Official Build) dev (64-bit)
 -  `OS: *` Linux
 -  `Stack Traces:`

**Comment by clshortfuse** [comment]
Good news! We have already fixed this in the master branch with this PR: https://github.com/angular/material/pull/9228
